framework/src/meshgenerators/DistributedRectilinearMeshGenerator.C

```diff
@@ -51,6 +51,14 @@ DistributedRectilinearMeshGenerator::validParams()
   params.addParam<Real>("ymax", 1.0, "Upper Y Coordinate of the generated mesh");
   params.addParam<Real>("zmax", 1.0, "Upper Z Coordinate of the generated mesh");
 
+  // The number of cores for the graph partition can be different from that used
+  // for mesh generation and simulation. Partitioners often become inefficient in
+  // compute time when the number of cores is large (around 10,0000).
+  // A possible "fix" is to partition the graph using a small number of cores
+  // when the mesh generation and the numerical simulation use a large number of processor cores.
```

Comment:
I think this comment could be more helpful as doxygen in the header

Reply:
Done